controller/k8s/clientset.go

```diff
@@ -16,10 +16,10 @@ func wrapTransport(config *rest.Config, telemetryName string) *rest.Config {
 	return config
 }
 
-// NewSpClientSet returns a Kubernetes ServiceProfile client for the given
+// NewL5DCRDClient returns a Linkerd controller client for the given
 // configuration.
-func NewSpClientSet(kubeConfig *rest.Config) (*spclient.Clientset, error) {
-	return spclient.NewForConfig(wrapTransport(kubeConfig, "sp"))
+func NewL5DCRDClient(kubeConfig *rest.Config) (*l5dcrdclient.Clientset, error) {
+	return l5dcrdclient.NewForConfig(wrapTransport(kubeConfig, "sp"))
```

Comment:
Should we call the `telemetryName` something else other than `sp`. Now that we also seem to read multiple items?